pubspec.yaml

```diff
@@ -1,13 +1,14 @@
 name: airship_flutter
-description: "The Airship flutter plugin. This package implements a cross-platform plugin interface to Airship's iOS and Android native SDKs. This allows core Airship functionality to be implemented by Flutter apps written in dart. The available core Airship functionality currently includes: push, in-app message, message center, actions, custom events and more."
-version: 5.5.0
+description: "The Airship flutter cross-platform plugin interface to Airship's iOS and Android native SDKs.
+Simplifies adding core Airship support to be Flutter dart apps."
```

Comment:
we could maybe shorten this to something like the following, assuming it meets the min/max requirements on pub.dev

> Cross-platform plugin interface for the native Airship iOS and Android SDKs. Simplifies adding Airship to Flutter apps.

Reply:
I like that